src/System.Security.Permissions/ref/System.Security.Permissions.csproj

```diff
@@ -20,6 +20,8 @@
     <Reference Include="System.Security" />
     <Reference Include="System.ServiceProcess" />
     <Reference Include="System.Transactions" />
+    <Reference Include="WindowsBase" />
```

Comment:
nit: ordering, same below

Reply:
I don't have enough sense of the package graphs to know if this dependency makes sense or not.  It sort of feels like a System.Xaml.Permissions package might be justified... but if Eric thinks it's okay it's fine with me.

Reply:
It’s ok. It’s a pure facade on desktop and dependencies don’t matter since desktop is a monolith.  On core dependencies do not change.